src/core/lib/resource_quota/arena.cc

```diff
@@ -21,18 +21,23 @@
 #include <atomic>
 #include <new>
 
+#include "absl/log/log.h"
+
 #include <grpc/support/alloc.h>
 #include <grpc/support/port_platform.h>
 
 #include "src/core/lib/resource_quota/resource_quota.h"
 #include "src/core/util/alloc.h"
+namespace grpc_core {
 
 namespace {
 
-void* ArenaStorage(size_t initial_size) {
+void* ArenaStorage(size_t& initial_size) {
   static constexpr size_t base_size =
-      GPR_ROUND_UP_TO_ALIGNMENT_SIZE(sizeof(grpc_core::Arena));
+      GPR_ROUND_UP_TO_ALIGNMENT_SIZE(sizeof(Arena));
   initial_size = GPR_ROUND_UP_TO_ALIGNMENT_SIZE(initial_size);
+  initial_size = std::max(initial_size,
```

Comment:
Given this use of max, should we change where we seed the initial size for the first call such that we add `arena_detail::BaseArenaContextTraits::ContextSize()` to that number?  Otherwise, the size for the first call will always be too small.

Reply:
The size for the first call is already way off... making more precise wrong math won't help.